openpype/tools/settings/settings/widgets.py

```diff
@@ -612,11 +612,39 @@ def mouseReleaseEvent(self, event):
         super(ProjectListView, self).mouseReleaseEvent(event)
 
 
+class ProjectListSortFilterProxy(QtCore.QSortFilterProxyModel):
+
+    def __init__(self, *args, **kwargs):
+        super(ProjectListSortFilterProxy, self).__init__(*args, **kwargs)
+        self._enable_filter = True
+
+    def filterAcceptsRow(self, source_row, source_parent):
+        if not self._enable_filter:
+            return True
+
+        index = self.sourceModel().index(source_row, 0, source_parent)
+        is_active = bool(index.data(self.filterRole()))
+        is_selected = bool(index.data(ProjectListWidget.ProjectSelectedRole))
```

Comment:
I quite don't get why selection is important? All projects should disappear when checkbox change value...

Reply:
This was for keeping current selected project visible in list when it just got deactivated.

Since the only way for user to know which project is being edited is from the selection of that list view, so I thought it would be better not to hide the project that just got deactivated from list as soon as the "Save" is clicked and processed, until the user changed the selection to another project settings.